components/layout/context.rs

```diff
@@ -91,6 +95,9 @@ pub struct SharedLayoutContext {
     /// Interface to the font cache thread.
     pub font_cache_thread: Mutex<FontCacheThread>,
 
+    /// A channel used to send the new animations back to layout.
+    pub new_animations_sender: Mutex<Sender<Animation>>,
```

Comment:
Wait, what's this about? Isn't the whole point to move this to the local context?